tests/suites/0_stateless/02_function/02_0002_function_others.sql

```diff
@@ -1,2 +1,12 @@
+SELECT '=== IGNORE ===';
 select count() from numbers(100) where ignore(number + 1);
 select count() from numbers(100) where not ignore(toString(number + 3), 1, 4343, 4343, 'a');
+SELECT '=== INET_ATON ===';
+SELECT INET_ATON('10.0.5.9');
+SELECT INET_ATON(NULL);
+SELECT INET_ATON('hello');
```

Comment:
Should we throw the exceptions if parsing failed? 
We can have `TRY_INET_ATON` later to enable the catch of the error to be null.